### ISSUE TITLE
Android password policy "Require alphanumeric value" redundant

### ISSUE THREAD

**Issue by n-jay** [wso2/product-iots#1627]
**Description:**
When creating a new password policy for Android there is an option to select or deselect the requirement for an alphanumeric value. 
![screenshot from 2018-01-24 11-26-14](https://user-images.githubusercontent.com/24616975/35316812-9af02462-00f9-11e8-957f-70e8dbeadba1.png)
Since Android password creation by default requires an alphanumeric value, the activation or deactivation of the "Require alphanumeric value" rule has no effect.
![screenshot from 2018-01-24 11-27-14](https://user-images.githubusercontent.com/24616975/35316822-b147d016-00f9-11e8-883b-a3e2e527a6fc.png)

**Suggested Labels:**
Type/Bug

**Suggested Assignees:**
@inoshperera 

**Affected Product Version:**
wso2iot-3.1.0-update11

**OS, DB, other environment details and versions:**    
Tested on Ubuntu 16.04 LTS

**Steps to reproduce:**
Apply policies before and after applying "Require alphanumeric value" rule.

**Comment by n-jay** [comment]
This problem persists for "Allow simple value" rule as well

**Comment by ruwany** [comment]
Although require alphanumeric chars property is redundant, allow simple value property seems to be in effect since after disabling said property, it would be required for users to enter a password containing a symbol character.

**Comment by n-jay** [comment]
@ruwany yes that's true.
I tested it out again before and after activating the "Allow simple value" rule. 
What I noted was that before activating we can get away with a password such as **1234a**. But upon activating we only need to enter an additional complex character to the end similar to this: **1234a@**.
But then again that conflicts with the **Minimum number of complex characters** selection, which in this case I have selected as none.

All in all, this is what the policy rules look like.
![screenshot from 2018-01-26 11-20-46](https://user-images.githubusercontent.com/24616975/35426714-0428052a-028b-11e8-9247-5358f5e49d15.png)
